src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -2390,6 +2392,18 @@ private void HandleReferences(RavenTransaction tx)
             _updateReferenceLoadWarning = false;
         }
 
+        private void HandleMismatchedReferences()
+        {
+            if (CurrentIndexingScope.Current._mismatchedReferencesHandler.MismatchedReferences.Count == 0) 
+                return;
+            
+            MismatchedReferencesLoadWarning warning = new (Name, CurrentIndexingScope.Current._mismatchedReferencesHandler.MismatchedReferences);
+
+            DocumentDatabase.NotificationCenter.Indexing.AddWarning(warning);
+                
+            CurrentIndexingScope.Current._mismatchedReferencesHandler.MismatchedReferences = null;
```

Comment:
Instead `.MismatchedReferences = null` you can introduce `Clear()` method

Reply:
Done